File: app/src/main/java/humbleactivity/app/DummyEffectorService.java
package humbleactivity.app;

import rx.Observable;

import java.util.Arrays;
import java.util.List;

class DummyEffectorService implements EffectorService {
    @Override
    public Observable<List<Filter>> listFilters() {
        return Observable.just(Arrays.asList(new Filter("Reverb"), new Filter("Distortion"), new Filter("Chorus"), new Filter("Delay")));
    }
}

File: app/src/main/java/humbleactivity/app/data/DummyEffectorService.kt
package humbleactivity.app.data

import rx.Observable
import java.util.concurrent.TimeUnit

internal class DummyEffectorService : EffectorService {
    override fun listFilters(): Observable<List<Filter>> {
        return Observable.just(listOf(Filter("Reverb"), Filter("Distortion"), Filter("Chorus"), Filter("Delay"))).delay(1, TimeUnit.SECONDS)
    }
}
